core/openapi.yaml

```diff
@@ -457,24 +486,59 @@ components:
             type: array
             minItems: 1
             items:
-              $ref: '#/components/schemas/Waypoint'
+              $ref: "#/components/schemas/Waypoint"
           example:
-            -
-              - track_section: "track_departure"
+            - - track_section: "track_departure"
                 direction: "START_TO_STOP"
                 offset: 4.2
               - track_section: "track_departure_bis"
                 direction: "START_TO_STOP"
                 offset: 14.2
-            -
-              - track_section: "track_destination"
+            - - track_section: "track_destination"
                 direction: "STOP_TO_START"
                 offset: 138.32
+    STDCMRequest:
+      properties:
+        infra:
+          type: string
+          example: "infraID"
+        rolling_stocks:
+          type: array
+          minItems: 1
+          items:
+            $ref: "#/components/schemas/RollingStock"
+        route_occupancies:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: Route ID
+              start_occupancy_time:
+                type: number
+                format: double
+              end_occupancy_time:
+                type: number
+                format: double
+              start_time:
+                type: number
+                format: double
+              start_points:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Waypoint"
+                  minimum: 1
+              end_points:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Waypoint"
+                  minimum: 1
```

Comment:
I'm not sure I get it, what's that?

Reply:
Start and end points for the pathfinding, it's the same as for the normal pathfinding but without intermediate points